src/ws/main.c

```diff
@@ -45,13 +45,15 @@ static gint      opt_port         = 9090;
 static gchar     *opt_address     = NULL;
 static gboolean  opt_no_tls       = FALSE;
 static gboolean  opt_local_ssh    = FALSE;
+static gchar     *opt_local_session = NULL;
 static gboolean  opt_version      = FALSE;
 
 static GOptionEntry cmd_entries[] = {
   {"port", 'p', 0, G_OPTION_ARG_INT, &opt_port, "Local port to bind to (9090 if unset)", NULL},
   {"address", 'a', 0, G_OPTION_ARG_STRING, &opt_address, "Address to bind to (binds on all addresses if unset)", NULL},
   {"no-tls", 0, 0, G_OPTION_ARG_NONE, &opt_no_tls, "Don't use TLS", NULL},
   {"local-ssh", 0, 0, G_OPTION_ARG_NONE, &opt_local_ssh, "Log in locally via SSH", NULL },
+  {"local-session", 0, 0, G_OPTION_ARG_STRING, &opt_local_session, "Launch a bridge in the local session", NULL },
```

Comment:
Please add a note here that this is supposed to be called with a `BRIDGE` argument (the man page already has this).

Reply:
Indeed, done. Also added it to `--address`.

Pushed that, plus the implied `--no-tls`, and adjusting the tests accordingly.